codegenerator/cli/src/hbs_templating/codegen_templates.rs

```diff
@@ -501,7 +517,10 @@ let eventSignatures = [{}]"#,
                     ))?;
 
                 format!(
-                    "let abi = %raw(`null`)\n{}\n{}",
+                    "let abi = Fuel.transpileAbi(%raw(`require(\"../../{}\")`))\n{}\n{}",
```

Comment:
I'm always nervous to see hardcoded relative paths. Should this not take some sort of parameter that's been parsed. Like generated relative path?

Reply:
I'm pretty sure even our integration tests would have this case.

Reply:
This is how `indexer-fuel` works. I'll cut the corner here, to make it work and then fix some of the issues. I already have 4 similar things on my list